packages/lesswrong/components/common/ToggleSwitch.tsx

```diff
@@ -6,17 +6,15 @@ const styles = (theme: ThemeType) => ({
   root: {
     cursor: "pointer",
     position: "relative",
-    width: 34,
-    minWidth: 34,
-    height: 20,
+    width: 28,
```

Comment:
All these changes affect the onboarding flow as well. Maybe we need variants?

Before:
<img width="232" alt="Screenshot 2024-02-26 at 11 34 29" src="https://github.com/ForumMagnum/ForumMagnum/assets/5075628/93047088-a3ad-4537-8cb5-febec552d547">

After:
<img width="239" alt="Screenshot 2024-02-26 at 11 33 02" src="https://github.com/ForumMagnum/ForumMagnum/assets/5075628/ee41fbd4-0868-4e2d-89a0-1f8bc90a13d6">

Reply:
ahh I didn't realise that, let me double check–maybe the small one looks fine in onboarding too

Reply:
yeah seems reasonable to have two different sizes, it does look very small in onboarding (but too big in the dropdown otherwise)

Reply:
don't think it's _crazy_ small in onboarding though, we could use the small one for now too if it's annoying creating two sets?

Reply:
Sounds good - I've made them use two different versions 👍